Website/src/rendering/src/models/ordercloud/DOrder.ts

```diff
@@ -8,3 +8,9 @@ export interface DOrderXp {
   // TODO: Export these magic strings as constants and use the constants in the code that use this property.
   DeliveryType?: 'PickupFromSummit' | 'PickupInStore' | 'Ship';
 }
+
+export const DELIVERY_TYPES = {
+  pickupFromSummit: 'PickupFromSummit',
+  pickupInStore: 'PickupInStore',
+  ship: 'Ship',
+};
```

Comment:
Feels like an enum would be a better fit in this case, for example something like this 

```
export enum DeliveryTypes {
  PickupFromSummit = 'PickupFromSummit',
  PickupInStore = 'PickupInStore',
  Ship = 'Ship',
}
```

and then use this instead of the magic strings like 'Ship'.